virttest/libvirt_xml/vm_xml.py

```diff
@@ -923,13 +923,45 @@ def get_cpu_topology(self):
             logging.debug("<cpu>/<topology> xml element not found")
         return topology
 
-    def get_disk_all(self):
+    def get_disk_all(self, disk_type=None, disk_device=None, reverse=False):
```

Comment:
I can understand one more parameter disk_device benefit this function, but  I don't think disk_type is beneficial to this function ,rather than make it more complex.

Reply:
Looking document https://libvirt.org/formatdomain.html#hard-drives-floppy-disks-cdroms, there are also different disk types.
I'm thinking if somebody only need disks of one type, like 'network', this parameter can help to do that. This is my option.
If all you don't think it will help, I can remove it. Thanks.